src/dali_model.h

```diff
@@ -254,6 +259,10 @@ class DaliModel : public ::triton::backend::BackendModel {
 
   void ReadPipelineProperties() {
     int config_max_batch_size = ReadMaxBatchSize(model_config_);
+    if (config_max_batch_size == 0) {
```

Comment:
and if it's 1? Is it still batched?

Reply:
I believe so. After all, `batch_size=1` seems like a valid configuration.